README.md

```diff
@@ -108,4 +139,12 @@ Core contains the controller class which inherites from the SQLiteBackend class
 
 The controller class also includes all the methods from the Employee and Customer classes.
 
-The purpose of the controller class is to mainly connect to the SQLiteBackend class and use the session as well as link the Employee and Customer classes to perform the functions.
\ No newline at end of file
+The purpose of the controller class is to mainly connect to the SQLiteBackend class and use the session as well as link the Employee and Customer classes to perform the functions.
+
+### server.py
+
+Server contains the controllor object which connects to the DB through bootstrapping. It is also responsible for creating and running the server.
+
+The employee and customer routes are mapped to their respective functions and the functions call the methods from the Controller class.
+
+The functions in server.py perform a GET, POST, PUT or DELETE command and displays the results in JSON format.
```

Comment:
The functions in server.py perform GET, POST, PUT and DELETE requests over HTTP REST and returns results in JSON format.

Reply:
change made